aten/src/ATen/native/cuda/UpSample.cuh

```diff
@@ -163,6 +163,15 @@ __device__ __forceinline__ static int nearest_neighbor_compute_source_index(
   return src_index;
 }
 
+__device__ __forceinline__ static int nearest_neighbor_compute_destination_index(
+    const float scale,
+    int src_index,
+    int output_size) {
+  const int dst_index = min<int>(
+      static_cast<int>(ceilf(src_index / scale)), output_size - 1);
```

Comment:
Wondering if precomputing invscale as 1/scale and using multiply here would speed things up?

Reply:
This is obsolete code. Will remove :)